exec.go

```diff
@@ -114,8 +114,8 @@ func execProcess(context *cli.Context) (int, error) {
 	if err != nil {
 		return -1, err
 	}
-	if status == libcontainer.Stopped {
-		return -1, errors.New("cannot exec a container that has stopped")
+	if status == libcontainer.Stopped || status == libcontainer.Paused {
+		return -1, fmt.Errorf("cannot exec in a %s container", status)
```

Comment:
I understand the argument for blocking this, but I do wonder if it's really runc's job to say that trying to execute a program in a paused container won't work until you unpause it? I can imagine few hypothetical scenarios where you might want to exec a program inside a container in a frozen state then unfreeze it, though I don't know how widely used this feature is.

Reply:
This comes from my own experience using runc. Execute `runc exec` and nothing happens. Do Ctrl-C and nothing happens.  Ctrl-Z does not help either. You have no idea what's going on, and that's super confusing even for me.

I do understand though that runc is not end user tool, and the scenario with "runc exec" followed by "runc resume" might be legit.

Not sure what to do here. Replace the rejection with a warning? This might mess with the output that a user expects. Add something like `--ignore-paused` flag? Does not look good either.

Drop this check entirely?

Reply:
(Separate) check for .Paused which tells user to use "--ignore-paused" flag to force running sounds fine for me.  Frozen invocations which do not accept Ctrl-C/-Z are really annoying even to experienced users.

Reply:
Not that bad solution, I guess (except that old runc will refuse this option, and a new one will require it for the frozen cgroup case).

Implemented, PTAL

Reply:
> except that old runc will refuse this option, and a new one will require it for the frozen cgroup case

A solution to that would be to use an environment variable, rather than a CLI option.
Something like `RUNC_EXEC_IGNORE_PAUSED`, which, if set, makes runc work as before.

Not sure it's all worth it.